packages/kit/types/ambient.d.ts

```diff
@@ -11,9 +11,12 @@
  * 	}
  * }
  *
- * export default undefined;
+ * export {};
  * ```
  *
+ * The `export {}` line ensure that this file is treated as a module so you can import other types into it. It means that you can't declare other ambient modules in it however.
+ * For this it's best to create another file next to it called `ambient.d.ts` and add your `declare module` statements there.
```

Comment:
> so you can import other types into it

I don't think this is quite right? The `export` doesn't exist so that you can `import`; the `export` just means that it's a module and not an ambient module _regardless_ of whether you `import`. I curse the TypeScript documentation for being so bad at articulating this stuff (there's literally nothing that I can find about what ambient module files are and how their behaviour differs, or about things like `declare global`), but in the absence of something more authoritative to link to I think this gets slightly closer to the truth?

```suggestion
 * The `export {}` line exists because without it, the file would be treated as an _ambient module_ which prevents you from adding `import` declarations.
 * If you need to add ambient `declare module` declarations, do so in a separate file like `src/ambient.d.ts`.
```